README.md

```diff
@@ -465,6 +465,33 @@ Here are the metrics settings:
     "vm.max_map_count" = "262144"
     ```
 
+#### Custom CA certificates settings
+
+By defualt, Bottlerocket ships with the Mozilla CA certificate store, but you can add self-signed certificates through the API using these settings:
+
+* `settings.pki.<bundle-name>.data`: Base64-encoded PEM-formatted certificates bundle; it can contain more than one certificate
+* `settings.pki.<bundle-name>.trusted`: Whether the certificates in the bundle are trusted; defaults to `false` when not provided
+
+Here's an example of adding a bundle of self-signed certificates, using API calls:
+
+```sh
+apiclient set \
+  pki.my-trusted-bundle.data="W3N..." \
+  pki.my-trusted-bundle.trusted=true  \
+  pki.dont-trust-these.data="N3W..."  \
+  pki.dont-trust-there.trusted=false
```

Comment:
We don't provide very many `apiclient set` examples elsewhere. Are we leading with this approach because we expect the size of certificate bundles to exceed what's practical to pass through user-data? That might be worth a comment if so.

Reply:
I didn't have any intention to lead users to use `apiclient`, but you made a great point and I'll document what could happen if users send bundles through user data.